### PR TITLE
document_requirements_for_classified_tiles

### PR DESCRIPTION
**Pull request by sandeepgadhwal**
this fixes https://github.com/ArcGIS/geosaurus/issues/2346

documented bit depth requirement of classified_tiles

### CODE REVIEW

File: guide/14-deep-learning/pixel_based_classification.ipynb [hunk 1]
```diff
@@ -79,14 +79,20 @@
    "cell_type": "markdown",
```
Comment (priyankatuteja) [inline guide/14-deep-learning/pixel_based_classification.ipynb]: <blockquote>Pre process training&nbsp;data that will be used for training data export</blockquote><p>I think we need to remove the space in Pre process</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (priyankatuteja) [inline guide/14-deep-learning/pixel_based_classification.ipynb]: <p>Notes:</p><ul><li>The raster and labeled imagery layer both are required to be '8 bit Unsigned' when the raster is a RGB layer such as world imagery.</li><li>The Labeled imagery layer should be a thematic raster with pixel values corresponding to the label class value.</li><li>The pixel values should range <strong>from from</strong> 1 to n, where n is the total number of classes.</li><li>Any <strong>nodata</strong> value should be mapped to 0, as portions of image with nodata values would be ignored in while exporting <strong>trianing</strong> data.</li></ul><p>Comment: </p><ul><li>The second bullet has redundant word 'from', last bullet has incorrect spelling which is in bold. </li><li>I am not sure if nodata should be no data, is it a specific keyword used?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Comment by priyankatuteja** [comment]
@sandeepgadhwal  I have suggested some changes in this file. I also found some more spell errors in the notebook, if you could fix them too. 

> Specify a folder name in raster store that will be used to store our training data
> Make sure a raster store is ready on your raster analytics image server. This is **where where** the output **subimages**, also called chips, labels and metadata files are going to be stored.
> Export training data using arcgis.learn
> With the feature class and raster layer, we are now ready to export training data using the export_training_data() method in arcgis.learn module. In **addtion** to feature class, raster layer, and output folder, we also need to **speficy** a few other parameters such as tile_size (size of the image chips), strid_size (distance to move each time when creating the next image chip), chip_format (TIFF, PNG, or JPEG), metadata format (how we are going to store those training labels). More detail can be found here.
> 
> Depending on the size of your data, tile and stride size, and computing resources, this **opertation** can take 15mins~2hrs in our experiment. Also, do not re-run it if you already run it once unless you would like to update the setting.
> 
> We will be using U-net, one of the well-**recogonized** image segmentation algorithm, for our land cover classification. U-Net is designed like …
> 
> As we can see, with only 10 epochs, we are already seeing reasonable results. Further **improvment** can be **acheived** through more sophisticated hyperparameter tuning. Let's save the model for further training or inference later. The model should be saved into a models folder in your folder. By default, it will be saved into your data_path that you specified in the very beginning of this notebook.

The bold ones are spelling mistakes.

**Comment by sandeepgadhwal** [comment]
Thanks @priyankatuteja , i will try to correct them as well.

File: guide/14-deep-learning/pixel_based_classification.ipynb (hunk 1 above)
Comment (sandeepgadhwal) [reply guide/14-deep-learning/pixel_based_classification.ipynb]: <p>Done</p>

Comment (sandeepgadhwal) [reply guide/14-deep-learning/pixel_based_classification.ipynb]: <p>i have changes nodata to NoData</p><p><a href="https://desktop.arcgis.com/en/arcmap/10.3/manage-data/raster-and-images/nodata-in-raster-datasets.htm" target="_blank">https://desktop.arcgis.com/en/arcmap/10.3/manage-data/raster-and-images/nodata-in-raster-datasets.htm</a></p>

**Comment by AtmaMani** [comment]
Thanks @sandeepgadhwal and @priyankatuteja
